Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+    <PropertyGroup>
+        <SentryOrg>sentry-sdks</SentryOrg>
```

Comment:
We should probably remove these...

Reply:
yeah prob good to add also some comments about where to get it. Maybe link to docs